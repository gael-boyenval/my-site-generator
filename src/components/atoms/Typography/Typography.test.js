/* eslint-disable no-undef */
import React from 'react';

import {
  BadassTitle,
  EditorialTitle,
  LeadBodyCopy,
  LeadBodyCopyEditorial,
  TinyHeading,
  Mention,
  SubTitle,
  Title3,
  Title3Sans,
} from './index';

describe('Component Typography BadassTitle', () => {
  it('should be render without crash', () => {
    const component = shallow(<BadassTitle>Hello</BadassTitle>);

    expect(toJson(component)).toMatchSnapshot();
  });
});

describe('Component Typography EditorialTitle', () => {
  it('should be render without crash', () => {
    const component = shallow(<EditorialTitle>Hello</EditorialTitle>);

    expect(toJson(component)).toMatchSnapshot();
  });
});

describe('Component Typography SubTitle', () => {
  it('should be render without crash', () => {
    const component = shallow(<SubTitle>Hello</SubTitle>);

    expect(toJson(component)).toMatchSnapshot();
  });
});

describe('Component Typography LeadBodyCopy', () => {
  it('should be render without crash', () => {
    const component = shallow(<LeadBodyCopy>Hello</LeadBodyCopy>);

    expect(toJson(component)).toMatchSnapshot();
  });
});
describe('Component Typography LeadBodyCopyEditorial', () => {
  it('should be render without crash', () => {
    const component = shallow(<LeadBodyCopyEditorial>Hello</LeadBodyCopyEditorial>);

    expect(toJson(component)).toMatchSnapshot();
  });
});

describe('Component Typography TinyHeading', () => {
  it('should be render without crash', () => {
    const component = shallow(<TinyHeading>Hello</TinyHeading>);

    expect(toJson(component)).toMatchSnapshot();
  });
});

describe('Component Typography Mention', () => {
  it('should be render without crash', () => {
    const component = shallow(<Mention>Hello</Mention>);

    expect(toJson(component)).toMatchSnapshot();
  });
});

describe('Component Typography Title3', () => {
  it('should be render without crash', () => {
    const component = shallow(<Title3>Hello</Title3>);

    expect(toJson(component)).toMatchSnapshot();
  });
});

describe('Component Typography Title3Sans', () => {
  it('should be render without crash', () => {
    const component = shallow(<Title3Sans>Hello</Title3Sans>);

    expect(toJson(component)).toMatchSnapshot();
  });
});
