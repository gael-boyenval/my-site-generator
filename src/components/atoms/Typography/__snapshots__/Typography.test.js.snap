// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Typography BadassTitle should be render without crash 1`] = `
.c0 {
  font-size: 7.2rem;
  font-family: "Bw Modelica","Helvetica Neue",Helvetica,Arial,sans-serif;
  font-weight: 900;
  line-height: 1;
  text-transform: uppercase;
  margin: 0;
  -webkit-transform: translateY(-0.3rem);
  -ms-transform: translateY(-0.3rem);
  transform: translateY(-0.3rem);
}

<h1
  className="c0"
>
  Hello
</h1>
`;

exports[`Component Typography EditorialTitle should be render without crash 1`] = `
.c0 {
  font-size: 6rem;
  font-family: "Gauthier FY","Palatino Linotype",Palatino,Palladio,"URW Palladio L","Book Antiqua",Baskerville,"Bookman Old Style","Bitstream Charter","Nimbus Roman No9 L",Garamond,"Apple Garamond","ITC Garamond Narrow","New Century Schoolbook","Century Schoolbook","Century Schoolbook L",Georgia,serif;
  font-weight: 500;
  line-height: 1;
  margin: 0;
}

<h1
  className="c0"
>
  Hello
</h1>
`;

exports[`Component Typography LeadBodyCopy should be render without crash 1`] = `
.c0 {
  font-size: 2.4rem;
  font-family: "Gauthier FY","Palatino Linotype",Palatino,Palladio,"URW Palladio L","Book Antiqua",Baskerville,"Bookman Old Style","Bitstream Charter","Nimbus Roman No9 L",Garamond,"Apple Garamond","ITC Garamond Narrow","New Century Schoolbook","Century Schoolbook","Century Schoolbook L",Georgia,serif;
  line-height: 1.5;
  margin: 0;
  -webkit-transform: translateY(-0.2rem);
  -ms-transform: translateY(-0.2rem);
  transform: translateY(-0.2rem);
}

<p
  className="c0"
>
  Hello
</p>
`;

exports[`Component Typography LeadBodyCopyEditorial should be render without crash 1`] = `
.c0 {
  font-size: 2rem;
  font-family: "Bw Modelica","Helvetica Neue",Helvetica,Arial,sans-serif;
  font-weight: bold;
  line-height: 1.5;
  margin: 0;
  -webkit-transform: translateY(0.1rem);
  -ms-transform: translateY(0.1rem);
  transform: translateY(0.1rem);
}

<p
  className="c0"
>
  Hello
</p>
`;

exports[`Component Typography Mention should be render without crash 1`] = `
.c0 {
  font-size: 1.6rem;
  font-family: "Bw Modelica","Helvetica Neue",Helvetica,Arial,sans-serif;
  font-weight: 500;
  line-height: 1.5;
  margin: 0;
  font-size: 1.2rem;
  margin: 0;
}

<p
  className="c0"
>
  Hello
</p>
`;

exports[`Component Typography SubTitle should be render without crash 1`] = `
.c0 {
  font-size: 3.6rem;
  font-family: "Gauthier FY","Palatino Linotype",Palatino,Palladio,"URW Palladio L","Book Antiqua",Baskerville,"Bookman Old Style","Bitstream Charter","Nimbus Roman No9 L",Garamond,"Apple Garamond","ITC Garamond Narrow","New Century Schoolbook","Century Schoolbook","Century Schoolbook L",Georgia,serif;
  font-weight: 500;
  line-height: 1.3333333333333333;
  margin: 0;
  -webkit-transform: translateY(0.1rem);
  -ms-transform: translateY(0.1rem);
  transform: translateY(0.1rem);
}

<h2
  className="c0"
>
  Hello
</h2>
`;

exports[`Component Typography TinyHeading should be render without crash 1`] = `
.c0 {
  font-size: 1.2rem;
  font-family: "Bw Modelica","Helvetica Neue",Helvetica,Arial,sans-serif;
  -webkit-letter-spacing: 0.15em;
  -moz-letter-spacing: 0.15em;
  -ms-letter-spacing: 0.15em;
  letter-spacing: 0.15em;
  text-transform: uppercase;
  font-weight: 600;
  line-height: 1;
  margin: 0;
}

<h4
  className="c0"
>
  Hello
</h4>
`;

exports[`Component Typography Title3 should be render without crash 1`] = `
.c0 {
  font-size: 3rem;
  font-family: "Gauthier FY","Palatino Linotype",Palatino,Palladio,"URW Palladio L","Book Antiqua",Baskerville,"Bookman Old Style","Bitstream Charter","Nimbus Roman No9 L",Garamond,"Apple Garamond","ITC Garamond Narrow","New Century Schoolbook","Century Schoolbook","Century Schoolbook L",Georgia,serif;
  line-height: 1.4;
  font-weight: 400;
  margin: 0;
  -webkit-transform: translateY(0.5rem);
  -ms-transform: translateY(0.5rem);
  transform: translateY(0.5rem);
}

<h3
  className="c0"
>
  Hello
</h3>
`;

exports[`Component Typography Title3Sans should be render without crash 1`] = `
.c0 {
  font-size: 2rem;
  font-family: "Bw Modelica","Helvetica Neue",Helvetica,Arial,sans-serif;
  line-height: 1.2;
  font-weight: bold;
  margin: 0;
}

<h3
  className="c0"
>
  Hello
</h3>
`;
